### PR TITLE
Add integration tests 

### LINKED ISSUE #241: Add integration tests
Like: https://github.com/sindresorhus/eslint-plugin-unicorn/tree/master/test/integration https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2a630e1f3242d4dc1bb654635f89fc06f1453d68/package.json#L17

We can just borrow that one.

### PR DESCRIPTION
**Pull request by gurrrung**
fixes #241

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,5 +4,11 @@ node_js:
   - 10
   - 8
   - 6
+matrix:
+  include:
+    - node_js: 11
```
Comment (sindresorhus) [inline .travis.yml]:
```suggestion
    - node_js: 12
```

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
https://travis-ci.org/avajs/eslint-plugin-ava/jobs/537169405 failing

**Comment by gurrrung** [comment]
It is trying to find `meta.type` in [bar.js](https://github.com/sindresorhus/globby/blob/master/fixtures/gitignore/bar.js), which is expected to not be there.

**Comment by sindresorhus** [comment]
> It is trying to find meta.type in bar.js , which is expected to not be there.

Huh? Why would it do that?

**Comment by sindresorhus** [comment]
@gurrrung Note that this is waiting for you to make Travis pass ;)

**Comment by gurrrung** [comment]
It's weird, how removing globby and clinton, makes the tests pass. @sindresorhus

**Comment by sindresorhus** [comment]
Yeah, I dunno either. Thanks :)

**Pull request merged**